frontend/src/lib/notify/index.ts

```diff
@@ -0,0 +1,38 @@
+import { writable } from 'svelte/store';
+export interface Notification {
+  id: string;
+  message: string;
+  category: string;
+  duration: number;
+}
+
+export const notifications = writable([]);
+
+export function addNotification(
```

Comment:
I'd like a few changes here that depend on each other to some extent:
1) We add/export the functions `notifySuccess`, `notifyWarning` etc. (maybe even just those 2 for now 🤷)
2) Those function use the full alert class names (e.g. `alert-success`)
3) Then you can remove the hidden div's you have in the notify component.
4) The `category` field in type `Notification` becomes more restrictive: `'alert-success' | `alert-warning``

Reply:
Sounds good, wouldn't I still need the hidden divs tho?

Reply:
As long as the complete css class names are [somewhere in our source files](https://tailwindcss.com/docs/content-configuration#configuring-source-paths) (*.ts or *.svelte), they'll get included, so you shouldn't need the divs.